rbac/management/utils.py

```diff
@@ -291,3 +292,55 @@ def get_admin_from_proxy(username, request):
 
     is_org_admin = bop_resp.get("data")[index]["is_org_admin"]
     return is_org_admin
+
+
+def validate_principal_type(queryParams):
+    """Validate the principal type.
+
+    Makes sure that the given "type" query parameter has a valid value, and returns it.
+
+    :param queryParams: The query parameters of the request.
+    :rtype: str
+    """
+    return validate_and_get_key(queryParams, "type", ["service-account", "user"], "user")
```

Comment:
Perfect, yes! Initially I did it that way but I was unsure about repeating the query parameters in two places. Since we're changing those query parameters for the group view, this simply goes away. Changed!

Reply:
That makes sense, thanks! 😄